dtschema/schemas/reserved-memory/reserved-memory.yaml

```diff
@@ -52,6 +52,26 @@ properties:
       Address and Length pairs. Specifies regions of memory that are
       acceptable to allocate from.
 
+  ecc-detection-bits:
+    default: 0
+    description: |
+      If present, this indicates the number of bits of memory error
+      which can be detected and reported by the Error-Correction Code
+      (ECC) memory subsystem (typically 0, 1 or 2).
+
+  ecc-correction-bits:
+    default: 0
+    description: |
+      If present, this indicates the number of bits of memory error
+      which can be corrected by the Error-Correction Code (ECC) memory
+      subsystem (typically 0, 1 or 2).
+
+  export:
```

Comment:
Shouldn't this be mutually exclusive with no-map and reusable?

Reply:
That's one of the things I wasn't too sure about. I don't think it's mutually exclusive with no-map per se. If we take the Linux example, we can totally create dma-buf from memory that isn't accessible to the CPU.

reusable however looks like it could be made mutually exclusive with it.